javaparser-core/src/main/java/com/github/javaparser/ast/Node.java

```diff
@@ -65,11 +62,13 @@
 	public Node() {
 	}
 
+	public Node(Position begin, Position end) {
+		this.begin = begin;
```

Comment:
Nah! Some people instanciate nodes manually without positions. The null checks are in the deprecated getter/setters.